podcasts/FeaturedSummaryViewController.swift

```diff
@@ -149,20 +150,58 @@ class FeaturedSummaryViewController: SimpleNotificationsViewController, GridLayo
             listType = delegate.replaceRegionName(string: title)
         }
 
+        let dispatchGroup = DispatchGroup()
+
+        var podcastsToShow: [DiscoverPodcast] = []
+
+        var sponsoredPodcastsToAdd: [Int: DiscoverPodcast] = [:]
+
+        dispatchGroup.enter()
         DiscoverServerHandler.shared.discoverPodcastList(source: source, completion: { [weak self] podcastList in
             guard let strongSelf = self, let discoverPodcast = podcastList?.podcasts else { return }
```

Comment:
We can remove the self references since it's not being used anymore.

```suggestion
        DiscoverServerHandler.shared.discoverPodcastList(source: source, completion: { podcastList in
            guard let discoverPodcast = podcastList?.podcasts else { return }
```